src/DataTypes/DataTypeString.h

```diff
@@ -21,6 +21,9 @@ class DataTypeString final : public IDataType
         return "String";
     }
 
+    // FIXME: string can contain arbitrary bytes, not only UTF-8 sequences
```

Comment:
Nothing to fix here.

But it raises a question, why we are using TEXT for AggregateFunction - it contains non-UTF8 sequences almost always.